operatorinterface.py

```diff
@@ -75,11 +75,16 @@ def __init__(self) -> None:
         controllerNumbers = set(
             i[0] for i in controlScheme.values()
         )  # set ensures no duplicates
+        print(f"Looking for controllers: {controllerNumbers} ...")
```

Comment:
Prints are a dangerous game. Unless these run on startup only, I don't recommend, especially for something like simulation when you would rarely fulfill the button expectation